### ISSUE TITLE
"Planning and Prerequisites Guide" is a link to RHV documentation

### ISSUE THREAD

**Issue by didib** [oVirt/ovirt-site#2641]
Perhaps we should have an oVirt "Planning and Prerequisites Guide" and link to it instead.

Perhaps also include something like https://access.redhat.com/articles/906543 .

**Comment by apinnick** [comment]
@sandrobonazzola  Who will write this guide? Can the existing downstream guide be adapted for oVirt?

**Comment by apinnick** [comment]
I can check but I am fairly certain that the RHV docs team does not have the resources for writing upstream docs. The P&P downstream guide has been moved to GitHub. If someone wants to create an oVirt version using the existing files, that might be the easiest way to create an oVirt-specific guide. I am going to unassign this from myself.

**Comment by sandrobonazzola** [comment]
marked as help wanted, looking for contribution from community for preparing this guide for oVirt.

**Comment by gocallag** [comment]
@sandrobonazzola isn't all this covered by https://www.ovirt.org/documentation/installing_ovirt_as_a_self-hosted_engine_using_the_command_line - eg. Chapter 2 Requirements ? Or did you have other thoughts?

**Comment by sandrobonazzola** [comment]
I think the issue here is that oVirt documentation is pointing to RHV documentation which may just disappear at some point.
I didn't check the content of the "Planning and Prerequisites Guide" but if it's already covered by installing guide best option would possibly be just dropping the link to the RHV documentation.

**Comment by gocallag** [comment]
yeah a quick check revealed 186 references to access.redhat, perhaps we use this ticket for that work. I'm happy to have a crack

**Comment by sandrobonazzola** [comment]
Closing as fixed via #3202
